draft-ietf-quic-transport.md

```diff
@@ -763,10 +763,12 @@ larger resource commitments at the receiver.  Thus there is a trade-off between
 resource commitment and overhead when determining how large a limit is
 advertised.
 
-A receiver MAY use an autotuning mechanism to tune the frequency and amount of
+A receiver can use an autotuning mechanism to tune the frequency and amount of
 advertised additional credit based on a round-trip time estimate and the rate at
 which the receiving application consumes data, similar to common TCP
-implementations.
+implementations.  Sending frames related to flow control only when there are
+other frames to send or when a peer is blocked ensures that flow control doesn't
```

Comment:
This sentence is very vague, and I'm not sure I find it useful. What issue is it tackling?

Reply:
It's an optimization I've suggested a few times to ensure sending more frequent updates doesn't have substantial overhead.

Reply:
Suggestion: "As an optimization, sending frames ..."